src/rastervision/common/models/densenet121.py

```diff
@@ -1,4 +1,13 @@
-# -*- coding: utf-8 -*-
+'''
+-*- coding: utf-8 -*-
```

Comment:
As a comment, it requires that the file be read by a certain python codec. I don't think it does any harm as it's only in the copied files.